docs/en/advanced_guides/detection_transformer.md

```diff
@@ -0,0 +1,230 @@
+# Detection Transformer
+
+Detection Transformers (DETRs) are a series of Transformer-based object detectors.
+
+In most DETRs, the features extracted from backbone and neck were fed into a Transformer which is composed of an encoder and a decoder. The Transformer directly outputs a set of queries in parallel. Each query corresponds to one prediction, which may be an object or `no object`.
+
+![DETR_overall](.\DETR_overall.png)
+
+There is a brand new implementation of DETRs in MMDetection, whose features are fourfold:
+
+#### 1. A unified base detector class for DETRs.
+
+According to the existing DETRs, a unified base detector class `DetectionTransformer` is designed to standardize the architectures and forward procedures of DETRs. (mmdet/models/detectors/base_detr.py)
```

Comment:
```suggestion
According to the existing DETRs, a unified base detector class [`DetectionTransformer`](../../../mmdet/models/detectors/base_detr.py) is designed to standardize the architectures and forward procedures of DETRs.
```

Reply:
OK~ Thanks very much for your review~ @RangeKing